core/src/main/java/org/apache/accumulo/core/spi/compaction/CompactionDirectives.java

```diff
@@ -43,6 +43,6 @@
   }
 
   public static Builder builder() {
-    return CompactionsDirectiveImpl.DEFAULT_BUILDER;
+    return CompactionDirectivesBuilder.DEFAULT_BUILDER;
```

Comment:
If there are two concurrent threads they would both be using the same object.

```suggestion
    return new CompactionDirectivesBuilder();
```